docs/installing.md

```diff
@@ -0,0 +1,100 @@
+```
+
+and in a second window:
+
+```
+./monerod --stagenet --no-igd --hide-my-port --data-dir node2 --p2p-bind-ip 127.0.0.1 --rpc-bind-ip 0.0.0.0 --confirm-external-bind --add-exclusive-node 127.0.0.1:48080 --rpc-login superuser:abctesting123 --rpc-access-control-origins http://localhost:8080
+```
+
+Now mine the first 130 blocks to a random address before using, so wallets only use the latest output type. For example, in a daemon: 
+
+```
+start_mining 56k9Yra1pxwcTYzqKcnLip8mymSQdEfA6V7476W9XhSiHPp1hAboo1F6na7kxTxwvXU6JjDQtu8VJdGj9FEcjkxGJfsyyah 1
+```
+
+## 4. Run Bitcoin
+
+Haveno still rely on Bitcoin for its infrastructure. We will remove it soon, in the meantime, to run Haveno locally, you'll need to download Bitcoin QT and run it in regtest mode. Which means you won't need to download the Bitcoin blockchain.
+
+1. Download and install [Bitcoin-Qt](https://bitcoin.org/en/download)
+2. Run Bitcoin-Qt in regtest mode, e.g.: `./Bitcoin-Qt -regtest -peerbloomfilters=1`
+3. Click on the *Console* tab and mine regtest bloks all at once. Run: `generatetoaddress 101 bcrt1q6j90vywv8x7eyevcnn2tn2wrlg3vsjlsvt46qz`
+
+## 5. Run Haveno
+
+Now that you have Both Bitcoin and Monero running, it's time to spin up your testing instance. You'll need to open 4 terminal windows: 1 for the seednode, 1 for the arbitrator/mediator, 1 for Alice (trader 1) and 1 for Bob (trader 2). Could be useful to rename the terminal, to have a better overview of what terminal belongs to whom (On Ubuntu: `echo -en "\033]0;NEW_TITLE\a"`).
+
+1. Run the seednode in one terminal:
+
+```
+./haveno-seednode --baseCurrencyNetwork=XMR_STAGENET --useLocalhostForP2P=true --useDevPrivilegeKeys=true --nodePort=2002 --appName=haveno-XMR_STAGENET_Seed_2002 --daoActivated=false
+```
+
+2. Run the Arbitrator instance:
+
+```
+./haveno-desktop --baseCurrencyNetwork=XMR_STAGENET --useLocalhostForP2P=true --useDevPrivilegeKeys=true --nodePort=4444 --appName=haveno-XMR_STAGENET_arbitrator --daoActivated=false --apiPassword=apitest --apiPort=9998
+```
+
+The (temporary) Haveno user interface will open. Once Haveno launches:
+
+3. Click on the *Account* tab and press `cmd+n`. Confirm the registration of the arbitrator.
+4. From the *Account* tab press `cmd+d` and confirm the registration of the mediator.
+
+Make sure to register arbitrator and mediator. Without them it's not possible to initiate a trade on the PoC.
+
+5. Run Alice
+
+```
+./haveno-desktop --baseCurrencyNetwork=XMR_STAGENET --useLocalhostForP2P=true --useDevPrivilegeKeys=true --nodePort=5555 --appName=haveno-XMR_STAGENET_Alice --daoActivated=false --apiPassword=apitest --apiPort=9999
+```
+
+6. Run Bob
+
+```
+./haveno-desktop --baseCurrencyNetwork=XMR_STAGENET --useLocalhostForP2P=true --useDevPrivilegeKeys=true --nodePort=6666 --appName=haveno-XMR_STAGENET_Bob  --daoActivated=false --apiPassword=apitest --apiPort=10000
+```
+
+## 5. Get yourself some stagenet coins
```

Comment:
Suggested rework of this section:

## Deposit Stagenet XMR to Alice and Bob

Alice and Bob's receiving address is printed to their terminal on startup.

Mining is the suggested method to receive new stagenet coins. The difficulty is locked to 10, so mining would be very fast. Go to your monerod instance and run `start_mining ADDRESS 1`. Monerod will start mining stagenet coins on your device using one thread. Replace `ADDRESS` with the address of Alice first, and then Bob's.

Alternatively, you can contact ErCiccione on Matrix (`@ErCiccione:haveno.network`) and give him a stagenet address.

Remember to send funds to both Alice and Bob.